jsonrpcbase/__init__.py

```diff
@@ -1,3 +1,3 @@
-from jsonrpcbase.main import JSONRPCService, DEFAULT_JSONRPC
+from jsonrpcbase.main import JSONRPCService
 
-__all__ = ['JSONRPCService', 'DEFAULT_JSONRPC']
```

Comment:
DEFAULT_JSONRPC was only needed in the tests, which seemed silly, so I unexported it